packages/redux/src/bags/__tests__/reducer.test.ts

```diff
@@ -153,7 +159,7 @@ describe('bags redux reducer', () => {
     });
 
     it('should handle other actions by returning the previous state', () => {
-      const state = { ...initialState, result: { bar: 'foo' } };
+      const state = { ...mockState.bag };
```

Comment:
Here we don't need the spread :) 

```suggestion
      const state = mockState?.bag;
```

Reply:
fixed